bin/node-gyp.js

```diff
@@ -20,6 +21,7 @@ var log = require('npmlog')
 var prog = gyp()
 var completed = false
 prog.parseArgv(process.argv)
+process.env['NODE_GYP_DIR'] = path.resolve(__dirname, '..');
```

Comment:
We should set this as a variable in `config.gypi`, rather than as an env variable.